src/components/Card.js

```diff
@@ -65,7 +65,8 @@ export const Card = ({ direction = 'ltr' }) => {
     }
 
     const ssid = escape(network.ssid);
-    const password = escape(network.password);
+    const password =
+      network.encryptionMode === 'None' ? '' : escape(network.password);
```

Comment:
Sorry, this was my mistake -- it should be checking for `nopass` instead of `None`

Reply:
Thanks, it's done.